web/modules/custom/lit_open_platform/src/Api/Client.php

```diff
@@ -242,7 +242,7 @@ protected function request(string $method, string $uri, array $options = []): mi
     try {
       $response = $client->request($method, $uri, $options);
 
-      $result = json_decode($response->getBody(), TRUE);
+      $result = json_decode($response->getBody(), TRUE, 512, JSON_THROW_ON_ERROR);
     }
     catch (ClientException $exception) {
```

Comment:
Maybe handle the exception throw by json_decode, so the end use will not get an 500